unit/java_bytecode/generate_concrete_generic_type/generic_field_array_instantiation.java

```diff
@@ -0,0 +1,11 @@
+public class generic_field_array_instantiation {
+
+    class generic<T> {
+        T gf;
+    }
+
+    generic<Float []> f;
```

Comment:
The logic to specialise these is moving to just specialise inputs for functions - you may want to modify this test to directly call the specialisation logic for the generic field.